path.go

```diff
@@ -52,18 +54,29 @@ func zoneFromPath(host string, path string, rec record) (string, int, error) {
 	return strings.Join(url, "."), from, nil
 }
 
-func getFinalRecord(zone string, from int) (record, error) {
+func getFinalRecord(zone string, from int, ctx context.Context, c Config) (record, error) {
 	var txts []string
 	var err error
 
-	txts, err = net.LookupTXT(zone)
+	if c.Resolver != "" {
+		net := net.Resolver{}
+		net.Dial(ctx, "tcp", c.Resolver)
+		txts, err = net.LookupTXT(ctx, zone)
+	} else {
+		txts, err = net.LookupTXT(zone)
+	}
 
 	// if nothing found, jump into wildcards
 	for i := 1; i <= from && len(txts) == 0; i++ {
 		zoneSlice := strings.Split(zone, ".")
 		zoneSlice[i] = "_"
 		zone = strings.Join(zoneSlice, ".")
 		txts, err = net.LookupTXT(zone)
+		if c.Resolver != "" {
+			net := net.Resolver{}
+			net.Dial(ctx, "tcp", c.Resolver)
+			txts, err = net.LookupTXT(ctx, zone)
+		}
```

Comment:
Seems like this would do 2 different lookups. One for the standard resolver and one with the custom one. Or am I missing something?

Reply:
We specify an alternate dialer on line 77 and use that instead of the default `net/http` package on line 78. Or at least that what I got from reading the `net/http`'s code.

Reply:
Right, but in line 73 you are still using the old net.LookupTXT().

Reply:
Ow, I missed that part and GitHub only shows line 76 to 79 :smile: my fault. will fix it right now.

Reply:
No worries. That's what reviews are for. Just thought it might have been intentional.